ci-operator/step-registry/ovn/sdn-migration/feature-migration/validate-egress-firewall/ovn-sdn-migration-feature-migration-validate-egress-firewall-ref.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"path": "ovn/sdn-migration/feature-migration/validate-egress-firewall/ovn-sdn-migration-feature-migration-validate-egress-firewall-ref.yaml",
+	"owners": {
+		"approvers": [
+			"sdn-approvers"
+		],
+		"reviewers": [
+			"sdn-reviewers"
+		]
+	}
+}
```

Comment:
nit: add newline

Reply:
This's a convention. If you check other `metadata.json` files, there is no newline at the end of the files either.